src/ServiceControl.AcceptanceTests/Recoverability/MessageFailures/When_processing_message_with_missing_metadata_failed.cs

```diff
@@ -89,30 +98,23 @@ protected override TransportOperations CreateMessage(MyContext context)
                     var headers = new Dictionary<string, string>
                     {
                         [Headers.MessageId] = context.MessageId,
-                        [Headers.ProcessingEndpoint] = context.EndpointNameOfReceivingEndpoint,
```

Comment:
Not having any error info should IMHO not be allowed. It was DLQ'd for a reason. It could be an processing error (exception) or another reason in the infrastructure..

THe SP UI is pretty useless for failures without errors as the grouping options would not work.

But for this test to pass it is indeed not required.

Reply:
Agreed, at least a message should be provided. Since the UI becomes useless without it, perhaps we should add it here?

Reply:
When ingesting the DLQ messages from ASB the only 2 things available is DLQ Reason and DLQ Description, I chose to map those like this:

NServiceBus.ExceptionInfo.ExceptionType: Dead letter reason
NServiceBus.ExceptionInfo.Message: Dead letter description


This leads to a bit of a funky UI experience since SP assumes a stack trace to be available and prints a `0` instead when displaying the grouping 
![image](https://github.com/user-attachments/assets/3de4fefc-4d71-49cb-8a39-9976ecd6501f)

When showing the actual message it looks better

![image](https://github.com/user-attachments/assets/3865ec29-12b0-4839-8096-0975b882528e)